app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -82,8 +82,22 @@ def republish_repositories
       respond_for_async :resource => task
     end
 
+    api :GET, "/content_view_versions/export_histories", N_("Show the export history for a content view version")
+    param :content_view_version_id, :number, :desc => N_("Content view version identifier"), :required => false
+    param :destination_server, String, :desc => N_("Destination Server name")
+    param :organization_id, :number, :desc => N_("Organization identifier")
```

Comment:
these two prams seem not-required either

Reply:
not required